### ISSUE TITLE
[UI idea] Inaccessible items (for keyboard users)

### ISSUE THREAD

**Issue by arjank** [Felienne/hedy#2925]
**Language selection and cheatsheet inaccessible for keyboard users**

The language selection menu and the cheatsheet are not accessible for keyboard users. You might be able to get focus to them (using the tab key) but it's not easy to see when an item has focus. And if those items have focus and you want to activate button using the space bar or the enter key, nothing happens. The items only appear when you hover over them using a mouse. (I have not tested tablets. They don't really have a hover state, so that might be an issue too.)

**Comment by TiBiBa** [comment]
Isn't this already fixed? We did a major re-structuring on the menubar to make it better align with the styling rules and also better accessible on touchscreen.

**Comment by AnneliesVlaar** [comment]
When using tab and spacebar the language switcher reacts
